packages/asc-ui/src/beta/components/Button/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './Button'
```

Comment:
Shouldn't this export also contain the named exports?

Reply:
Do you mean the current export should be named? So `export { default as Button } from './Button'`? Or that the named exports in `ButtonStyle.ts` should be exported?